### ISSUE TITLE
Panel probably deserves a mention?

### ISSUE THREAD

**Issue by dhirschfeld** [nbgallery/dashboards#1]
https://github.com/pyviz/panel

**Comment by somedave** [comment]
Awesome - thanks for the recommendation.  We hadn't seen Panel before.  We'll check it out and update our Readme.

**Comment by dhirschfeld** [comment]
I haven't actually given it a go myself but it does look interesting.

New blog post:  https://medium.com/@philipp.jfr/panel-announcement-2107c2b15f52

**Comment by jeffyjefflabs** [comment]
Yes, panel is very nice.  I'll add a link to the README and I'm also adding support in nb2dashboard.

**Comment by jeffyjefflabs** [comment]
cc @philippjfr

**Comment by jeffyjefflabs** [comment]
See dcde9f2313b29822e68e6fdf3de08f8e92129fe3
